templates/layout.html

```diff
@@ -6,8 +6,6 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>{% block title %}{% endblock %}</title>
-    <link rel="stylesheet" type="text/css" href="../static/css/bootstrap.min.css">

```

Comment:
On docker, everything is automated by Dockerfile. 

If you want to run the project manually using system installed stuff, that will be taken care of by the [shell script](https://github.com/fahdi/palettro/blob/c6ba5022ec31f7891438cd2f6ecfe355c850be10/build.sh).  You can see the commands and run them manually if you like.